src/LodeRunner/src/Program.cs

```diff
@@ -139,6 +142,7 @@ private static void ConfigureServicesAndSystemObjects(ServiceCollection services
                 .AddSingleton<CancellationTokenSource>(CancelTokenSource)
                 .AddSingleton<LodeRunnerService>()
                 .AddSingleton<ILodeRunnerService>(provider => provider.GetRequiredService<LodeRunnerService>());
+            Sdk.SetDefaultTextMapPropagator(new B3Propagator());
```

Comment:
OpenTelemetry Library may not be needed as this project is only used to create context and not propagate it

Reply:
Tested without the OTel library and verified that .NET's builtin Activity is in compliance with B3 Trace and Span format.

- Tested locally
- Tested with ngsa in k8s cluster and local LodeRunner